src/tools/csi-kata-directvolume/pkg/spdkrpc/jsonrpc.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2025 Kata Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package spdkrpc
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"time"
+)
+
+const SpdkErrNoDevice = -19
```

Comment:
IMHO, the error code corresponds to the Linux error code `ENODEV (19)`. Could we import it from the libc library? No sure if the go has such library. /cc @Apokleos